react/features/base/tracks/middleware.js

```diff
@@ -135,8 +135,13 @@ MiddlewareRegistry.register(store => next => action => {
             const isVideoTrack = jitsiTrack.type !== MEDIA_TYPE.AUDIO;
 
             if (isVideoTrack) {
-                if (jitsiTrack.isLocal()) {
-                    APP.conference.setVideoMuteStatus(muted);
+                if (jitsiTrack.type === MEDIA_TYPE.PRESENTER) {
+                    APP.UI.emitEvent(UIEvents.PRESENTER_MUTED, muted);
```

Comment:
What about calling `mutePresenter` directly instead of adding more to APP.UI, which is also legacy? Ideally not adding to conference.js would be nice but potentially not practical.

Reply:
Done.